dictionaries/nl.dictionary.itop.ui.php

```diff
@@ -985,24 +992,24 @@
 </ol>
 </p>
 <p>
-<i><b>Acties</b></i> define the actions to be performed when the triggers execute. For now there are only two kind of actions:
+<i><b>Acties</b></i> definieer de acties die uitgevoerd moeten worden als de trigger geactiveerd wordt. Er zijn 2 soorten standaardacties:
```

Comment:
```suggestion
<i><b>Acties</b></i> definieer de acties die uitgevoerd moeten worden als de triggers geactiveerd worden. Er zijn 2 soorten standaardacties:
```

Maybe also add "momenteel" as translation to "For now"?